ivadomed/maths.py

```diff
@@ -0,0 +1,40 @@
+import nibabel as nib
+import numpy as np
+import os
+import scipy
+import ivadomed.transforms as imed_transforms
+
+
+def gaussian_kernel(kernlen=10):
+    """
+    Create a 2D gaussian kernel with user-defined size.
+
+    Args:
+        kernlen (int): size of kernel
+
+    Returns:
+        ndarray: a 2D array of size (kernlen,kernlen)
+    """
+
+    x = np.linspace(-1, 1, kernlen + 1)
+    kern1d = np.diff(scipy.stats.norm.cdf(x))
+    kern2d = np.outer(kern1d, kern1d)
+    return imed_transforms.rescale_values_array(kern2d / kern2d.sum())
```

Comment:
`rescale_values_array` could be a good candidate for this file!!!

Reply:
Good idea !!

Reply:
done in 8aed599